entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/view/EntityPreview.java

```diff
@@ -0,0 +1,57 @@
+package eu.europeana.entitymanagement.definitions.view;
+
+import java.util.List;
+import java.util.Map;
+
+import eu.europeana.entitymanagement.definitions.model.search.results.IdBean;
+import eu.europeana.entitymanagement.definitions.model.vocabulary.EntityTypes;
+
+/**
+ * @author Sergiu Gordea
+ *
+ */
+public interface EntityPreview extends IdBean {
```

Comment:
this class is not used in entity management, please remove

Reply:
Removed.